service-front/app/src/Common/src/ConfigProvider.php

```diff
@@ -110,6 +110,7 @@ public function getDependencies(): array
                 SessionMiddleware::class                   => SessionMiddlewareFactory::class,
                 SessionExpiryMiddleware::class             => SessionExpiryMiddlewareFactory::class,
                 Middleware\I18n\SetLocaleMiddleware::class => Middleware\I18n\SetLocaleMiddlewareFactory::class,
+                Common\Middleware\ErrorHandling\GoneHandler::class => \Common\Middleware\ErrorHandling\GoneHandlerFactory::class,
```

Comment:
```suggestion
                Middleware\ErrorHandling\GoneHandler::class => Middleware\ErrorHandling\GoneHandlerFactory::class,
```

As the configprovider exists in the common namespace it's not necessary to repeat it.